src/main/resources/portal.properties.EXAMPLE

```diff
@@ -78,6 +78,9 @@ skin.right_nav.show_whats_new=true
 # settings controlling what to show in the right navigation bar
 skin.study_view.link_text=To build your own case set, try out our enhanced Study View.
 
+# setting controlling the blurb in study view
+skin.study_view.blurb=
```

Comment:
@zhx828 does it work when this one is commented out as well? Would prefer to not set default values in EXAMPLE properties file whenever possible

Reply:
@inodb it works as well. I commented out.